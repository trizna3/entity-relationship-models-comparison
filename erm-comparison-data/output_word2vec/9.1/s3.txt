airplane model-EMPTY
transport model-EMPTY
marine-soldiers
marine role-EMPTY
fight model-EMPTY
rocket type-rocket type
airplane-airplanes
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rocket type-fighter planes
GENERALIZATION_TO_11_ASSOCIATION:S-fighter planes-airplanes
CHANGE_CARDINALITY:S-fighter planes-airplanes
REBIND_1NN1_TO_MN:S-fighter planes
