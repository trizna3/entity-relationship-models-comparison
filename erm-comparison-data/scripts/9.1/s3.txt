entity set airplanes {
	"serial number"
	"name"
	"rocket count"
};

entity set "fighter planes" {
	"rocket count"
	"rocket type"
	"rocket type to shoot"
};

entity set "transport planes" {
	"person capacity"
	"load volume"
	"load weight"
};

entity set soldiers {
	"name"
	"actual health"
	"maximal health"
	"speed"
	"strength"
	"agility"
	"inteligence"
};

entity set roles {
	"role"
	"speed increase"
	"strength increase"
	"agility increase"
	"inteligence increase"
	"maximal health increase"
};

association  "actual soldier airplane" between
{
	airplanes MANY
	soldiers ONE
}{
	"airplane start"
};

association  between
{
	soldiers MANY
	roles ONE
}{};

generalization from "fighter planes" to "airplanes";

generalization from "transport planes" to "airplanes";
