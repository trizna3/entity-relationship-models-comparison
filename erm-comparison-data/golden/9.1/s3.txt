airplane model-EMPTY
fight model-fighter planes
rocket type-rocket type
transport model-transport planes
airplane-airplanes
marine-soldiers
marine role-roles
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rocket type-fighter planes